### DISCUSSION TITLE
Some feature requests from a international user.

### DISCUSSION THREAD

**Discussion by pzm9012** [linuxdeepin/developer-center#5313] [Features Request & Ideas | 特性请求 & 头脑风暴]
View https://bbs.deepin.org/post/261404

**Comment by BLumia** [comment]
Quoted from the post:

> there is missing component in Deepin like Print Manager Config there in no setting Print Manager Config on Control Center so I must search manually to find Print Manager on installed app.

Since the print manager is not a control-center plugin so...

It can be a control-center plugin, but it depends on the maintainers of the Print Manager project.

> And there is no option to setting Firewall Config on Control Center

There could be a control center plugin for managing firewall rules :)

> can Deepin add the feature like add-apt-repository under Control Center and all the stuff I mention in here on Deepin v23?

Although it's do-able, I am actually a little worried about the use-case of this feature. Currently v23 doesn't have many known 3rd-party PPAs, adding that right now might seem like "encourage" people to add random PPAs for other distros from the internet, which is discouraged in most cases.

cc @meiyixiang

**Reply by raspin0** [reply]
With PPA's on the verge of depreciation, linglong on the horizon and barely no PPAs directly made for deepin, I don't think this a way moving forward
